src/main/webapp/i18n/de/plagiarism.json

```diff
@@ -57,20 +58,27 @@
                 "plagiarismCases": "Plagiatsfälle",
                 "plagiarismCase": "Plagiatsfall",
                 "resolvedPlagiarismCase": "Gelöster Plagiatsfall",
+                "notification": {
+                    "title": "Plagiatsfall: {{ exercise }}",
+                    "body": "Hallo {{ student }}, nach einer sorgfältigen Durchsicht Deiner finalen Abgabe zur Übung “{{ exercise }}“ im Kurs “{{ course }}“ sind wir zu dem Schluss gekommen, dass Du ein Plagiat begangen hast. Dies ist ein Verstoß gegen die [Grundsätze guter studentischer Praxis]({{ cocLink }}) der Fakultät für Informatik, denen Du zugestimmt hast. Die *[Allgemeine Studien- und Prüfungsordnung]({{ aspoLink }})* regelt in §22.1 die Konsequenzen für solche Fälle. **Du hast eine Woche Zeit, um eine Erklärung abzugeben.**"
```

Comment:
This text is very specific only for TUM and even there only for the computer science department. I don’t think it should be this specific for a system that is used across multiple different universities.

Reply:
I just translated the text that has been used so far as a placeholder/default for the student notification. Instructors are free to change the text before submitting the post. Do you have a suggestion what it should read instead?

Reply:
Okay, I wasn’t aware that this is just a template and can be adapted manually as needed. In that case keeping it as-is is fine by me.